docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -543,8 +543,12 @@ To migrate content from one collection to another, if the collections are parts
 #. When the next major release comes:
 
   * remove the module/plugin from the source collection
-  * add ``redirect`` to the corresponding entry in ``meta/runtime.yml``
+  * remove its unit and integration tests, specific module utils, and documentation fragments if exist from the source collection. Be sure, they are not used by other modules/plugins and tests
```

Comment:
> Be sure, they are not used by other modules/plugins and tests

How can we be sure, is that only for `tests/integration/targets/setup_*` tests?

Reply:
Yes

Reply:
`grep` usually helps

Reply:
i'll reformulate this a little bit

Reply:
i removed this sentence, it's, imo, unnecessary